src/main/java/core/basesyntax/Bulldozer.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+public class Bulldozer extends Machine {
+
+    @Override
+    public void doWork() {

```

Comment:
```suggestion
public class Bulldozer extends Machine {
    @Override
    public void doWork() {
```
![image](https://github.com/mate-academy/jv-oop/assets/20326858/77182d38-a4f3-4313-8030-c89d431ad1db)